src/modules/clusters/cli/cluster.transformer.tsx

```diff
@@ -80,7 +82,7 @@ export const transformClusterData = (items, extra: any) => {
       return order.indexOf(a.status.text) - order.indexOf(b.status.text);
     });
   } catch (e) {
-    console.log(e);
+    logger.error(e);
```

Comment:
Add more descriptive explanation, showing that it was attempt to trnsform cluster data. not just raw error

Reply:
V